install.sh

```diff
@@ -194,94 +195,95 @@ __install_installer_deps() {
 	done
 }
 
-__install_bro() {
+__install_zeek() {
 
-
-	if  [ "$_OS" == "Ubuntu" ] && [ "$_OS_CODENAME" == "xenial" -o "$_OS_CODENAME" == "trusty" ] ; then
-		# Bro 2.6 will not compile under Debian 8/ Ubuntu Trusty/ Xenial in the OpenSuse Build Service
-		# Install Bro 2.5.5
-		__add_deb_repo "deb http://download.opensuse.org/repositories/home:/logan_bhis:/branches:/network:/bro/xUbuntu_$(lsb_release -rs)/ /" \
-			"Bro" \
-			"https://download.opensuse.org/repositories/home:logan_bhis:branches:network:bro/xUbuntu_$(lsb_release -rs)/Release.key"
+	if  [ "$_OS" == "Ubuntu" ] && [ "$_OS_CODENAME" == "xenial" ] ; then
+		# The Zeek Project does not host a package repo for Ubuntu 16.04.
+		# Use Security Onion's version of Zeek.
+		sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 9373E47F9BF8216D23D32BBBE1E6759023F386C7
+		__add_deb_repo "deb http://ppa.launchpad.net/securityonion/stable/ubuntu xenial main" \
+		"securityonion-ubuntu-stable-xenial"
+		__install_packages securityonion-bro
```

Comment:
I ran an install on xenial, and it doesn't look like the security onion package defines the necessary dependencies to run zeek. Running zeekctl on line 311 failed with `/usr/bin/env`.

The following dependencies are listed for the `securityonion-bro` package.
```
  Depends: libc6
  Depends: libc6
  Depends: libgcc1
  Depends: libjemalloc1
  Depends: libpython2.7
  Depends: libssl1.0.0
  Depends: libstdc++6
  Depends: securityonion-pfring-userland
  Depends: zlib1g
  Depends: gawk
  Depends: libcap2-bin
  Depends: securityonion-pfring-module
```

The zeek-lts package from 18.04 depends on the following:
```
  Depends: zeek-lts-core
    Depends: libpcap0.8
    Depends: libc6
    Depends: libc6
    Depends: libgcc1
    Depends: libssl1.1
    Depends: libstdc++6
    Depends: zlib1g
  Depends: zeekctl-lts
    Depends: python
```

It looks like we should just be able to add python to the install packages line. I'll look into the libpcap / pfring differences.

Reply:
The pfring libs seem to be correctly linked. I'll have to spin up a local VM to test if the packet capturing is actually working.

Reply:
After running Xenial in a VM, the capture appears to be working correctly with PF_RING.

Reply:
I'm not seeing the same failure using a fresh and updated Xenial VM. Querying `zeekctl cron` shows that it's enabled. If `zeekctl` still isn't working for you, let's chat about it.

Reply:
I ran the install on a new 16.04 VM as well. I was using the server version of Ubuntu and zeekctl failed because it couldn't find python. Even though I'm fairly sure python3 was installed and available. Anyway once I did `apt install python` it worked for me. I did `zeekctl deploy` and am getting logs.

Reply:
Done.